sci-electronics/fritzing/fritzing-0.9.2b-r1.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PARTS_P="${PN}-parts-${PV}"
+
+inherit qmake-utils
+
+DESCRIPTION="Electronic Design Automation"
+HOMEPAGE="http://fritzing.org/"
+SRC_URI="https://github.com/fritzing/fritzing-app/archive/${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/fritzing/fritzing-parts/archive/${PV}.tar.gz -> ${PARTS_P}.tar.gz"
+
+LICENSE="CC-BY-SA-3.0 GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="dev-qt/qtconcurrent:5
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtnetwork:5
+	dev-qt/qtprintsupport:5
+	dev-qt/qtserialport:5
+	dev-qt/qtsql:5[sqlite]
+	dev-qt/qtsvg:5
+	dev-qt/qtwidgets:5
+	dev-qt/qtxml:5
+	dev-libs/quazip[qt5]"
+DEPEND="${RDEPEND}
+	>=dev-libs/boost-1.40"
+
+S="${WORKDIR}/${PN}-app-${PV}"
+
+src_prepare() {
+	local translations=
+
+	# Get a rid of the bundled libs
+	# Bug 412555 and
+	# https://code.google.com/p/fritzing/issues/detail?id=1898
+	rm -rf src/lib/quazip/ pri/quazip.pri src/lib/boost* || die
+
+	# Fritzing doesn't need zlib
+	sed -i -e 's:LIBS += -lz::' -e 's:-lminizip::' phoenix.pro || die
+
+	# Somewhat evil but IMHO the best solution
+	for lang in $LINGUAS; do
```

Comment:
declare `lang` as local variable before.
Also consider switching the gettext `LINGUAS` variable to the modern Gentoo `L10N` handling scheme.